meta/SaiSerialize.cpp

```diff
@@ -1940,6 +1940,28 @@ static json sai_serialize_json_fdb_event_notification_data(
     return j;
 }
 
+std::string sai_serialize_nat_event(
+        _In_ sai_nat_event_t event)
+{
+    SWSS_LOG_ENTER();
+
+    return sai_serialize_enum(event, &sai_metadata_enum_sai_nat_event_t);
+}
+
+static json sai_serialize_json_nat_event_notification_data(
+        _In_ const sai_nat_event_notification_data_t& nat_event)
+{
+    SWSS_LOG_ENTER();
+
+    json j;
+
+    j["nat_event"] = sai_serialize_nat_event(nat_event.event_type);
```

Comment:
I was following similar to fdb to be consistent with naming.
j["fdb_event"] = sai_serialize_fdb_event(fdb_event.event_type);

If this is not a big deal, is it okay to keep the same to avoid the testing again.

Reply:
ok